From 1f05790ce62e3a0e38176e37afba87c7ddb3a92b Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Wed, 6 Aug 2014 03:47:05 -0400
Subject: [x86] pci: Reword comments

Message-id: <20140806034705.1438.7474.stgit@gir.stowe>
Patchwork-id: 87437
O-Subject: [RHEL7.1 PATCH 28/69] x86/PCI: Reword comments
Bugzilla: 1110896
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110896
Upstream Status: affbda86feef8da5056effe5d71da7981b2a9565

commit affbda86feef8da5056effe5d71da7981b2a9565
Author: Bjorn Helgaas <bhelgaas@google.com>
Date:   Fri Jan 24 10:38:40 2014 -0700

    x86/PCI: Reword comments

    Reword comments so they make sense.

    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/x86/pci/acpi.c b/arch/x86/pci/acpi.c
index efee6fc..9eeea64 100644
--- a/arch/x86/pci/acpi.c
+++ b/arch/x86/pci/acpi.c
@@ -509,15 +509,12 @@ struct pci_bus *pci_acpi_scan_root(struct acpi_pci_root *root)
  sd->domain = domain;
  sd->node = node;
  sd->acpi = device->handle;
- /*
-  * Maybe the desired pci bus has been already scanned. In such case
-  * it is unnecessary to scan the pci bus with the given domain,busnum.
-  */
+
  bus = pci_find_bus(domain, busnum);
  if (bus) {
   /*
-   * If the desired bus exits, the content of bus->sysdata will
-   * be replaced by sd.
+   * If the desired bus has been scanned already, replace
+   * its bus->sysdata.
    */
   memcpy(bus->sysdata, sd, sizeof(*sd));
   kfree(info);
-- 
1.7.1